ext/oci8/oci8.stub.php

```diff
@@ -239,15 +239,15 @@ function ocifetchinto($statement, &$result, int $mode = OCI_NUM): int|false {}
  * @param resource $statement
  * @param array $output
  */
-function oci_fetch_all($statement, &$output, int $offset = 0, int $limit = -1, int $flags = 0): int {}
+function oci_fetch_all($statement, &$output, int $offset = 0, int $limit = -1, int $flags = OCI_FETCHSTATEMENT_BY_COLUMN): int {}
```

Comment:
This should be `OCI_FETCHSTATEMENT_BY_COLUMN | OCI_ASSOC`, similar to current doc https://www.php.net/manual/en/function.oci-fetch-all.php#refsect1-function.oci-fetch-all-description.  Ditto for the other changes in this PR.